clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -73,6 +73,26 @@ def LangAttr : CIR_Attr<"Lang", "lang"> {
   let genVerifyDecl = 0;
 }
 
+//===----------------------------------------------------------------------===//
+// OptInfoAttr
+//===----------------------------------------------------------------------===//
+
+def CIR_OptInfoAttr : CIR_Attr<"OptInfo", "opt_info"> {
+  let summary = "A module-level attribute that holds the optimization level";
```

Comment:
`A module-level attribute that holds the optimization level` -> `A module-level attribute that holds optimization information`

Reply:
Updated.